components/molecule/accordion/src/index.scss

```diff
@@ -0,0 +1,102 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import './settings';
+
+.sui-MoleculeAccordion-tab {
+  background-color: $bgc-accordion-tab-container;
+  border: $bd-accordion-tab-container;
+  border-bottom: $bd-accordion-tab-container-none;
+  color: $c-accordion-tab-text;
+  display: flex;
+  flex-direction: column;
+  font-size: $fz-base;
+  margin: 0;
+  padding: 0;
+  position: relative;
+  width: 100%;
+
+  &:last-child {
+    border-bottom: $bd-accordion-tab-container;
+  }
+
+  &-btn {
```

Comment:
Hi @adriadoz, following the SUIT naming standard for descendent elements (https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#ComponentName-descendentName) it should be written in camel case. So it would be `&Btn`, the same for the following nested selectors.